samples/SampleStartups/SampleStartups.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net451;netcoreapp1.1</TargetFrameworks>
```

Comment:
Looking closer its neither, not supposed to be run, just showing examples of how startups work